dbclient/health/src/main/java/io/helidon/dbclient/health/DbClientHealthCheck.java

```diff
@@ -104,6 +283,64 @@ public Builder name(String name) {
             this.name = name;
             return this;
         }
+
+        /**
+         * Set health check statement type to query.
+         * Default health check statement type is DML.
+         *
+         * @return updated builder instance
+         */
+        public Builder query() {
+            this.isDML = false;
+            this.isNamedstatement = true;
+            return this;
+        }
+
+        /**
+         * Set custom statement name.
+         * Default statement name value is {@code ping}.
+         *
+         * @param name custom statement name.
+         * @return updated builder instance
+         */
+        public Builder statementName(String name) {
+            if (statement != null) {
+                throw new UnsupportedOperationException(
+                        "Can't use both statementName and statement methods in a single builder instance!");
+            }
+            this.isNamedstatement = true;
+            this.statementName = name;
+            return this;
+        }
+
+        /**
+         * Set custom statement.
+         *
+         * @param statement custom statement name.
+         * @return updated builder instance
+         */
+        public Builder statement(String statement) {
+            if (statementName != null) {
+                throw new UnsupportedOperationException(
+                        "Can't use both statementName and statement methods in a single builder instance!");
+            }
+            this.isNamedstatement = false;
+            this.statement = statement;
+            return this;
+        }
+
+        /**
+         * Set custom timeout to wait for statement execution response.
+         * Default value is {@code 10} seconds.
+         *
+         * @param seconds number of seconds to wait for statement execution response
+         * @return updated builder instance
+         */
+        public Builder timeout(int seconds) {
```

Comment:
When configuring timeout in code, please use 
`timeout(long duration, TimeUnit timeUnit)`
or you need to name the method to be clear (e.g. `timeoutSeconds`) - in this case I would prefer the first one
If there is support for it in config, the configurable value must be `timeout-seconds` or `timeout-millis` and the code that converts it calls the method with TimeUnit)

Reply:
changed to `public Builder timeout(long duration, TimeUnit timeUnit)`.